hail/python/hailtop/batch_client/aioclient.py

```diff
@@ -345,7 +345,7 @@ async def jobs(self, q=None):
                 break
 
     async def get_job_log(self, job_id: int):
-        return self._client.get_job_log(self.id, job_id)
+        return await self._client.get_job_log(self.id, job_id)
```

Comment:
noticed this while debugging.

Reply:
Nice catch. Could you add a return type annotation? Hopefully that can mitigate these bugs